### PR TITLE
feat: add button state

### LINKED ISSUE #9489: Mint button state when wallet is not connected
it's nice that it's checking, but let's put something more optimized there
image

Let's put "Connect your wallet" as a button label - clickable pink button
- after clicking, it will open the wallet sidebar 
- then it will check for the requirements and display the state we already have

### PR DESCRIPTION
**Pull request by pbkompasz**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [X] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [X] Closes #9489
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [X] My contribution builds **clean without any errors or warnings**
- [X] I've merged recent default branch -- **main** and I've no conflicts
- [X] I've tried to respect high code quality standards
- [X] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [X] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=16hqk8Tjugo9AwSzKkxXc1dsRK7iJetpx1YBaoXYwdHzuNAt&usdamount=50&donation=true)

#### Community participation

- [ ] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by exezbcz** [comment]
image
should be active - once clicked, the wallet connect sidebar is opened

File: components/collection/drop/MintButton.vue [hunk 1]
```diff
@@ -6,7 +6,7 @@
     no-shadow
     size="large"
     :loading="loading"
-    :disabled="buttonMint.disabled"
+    :disabled="buttonMint.disabled && !isWalletConnecting"
```
Comment (daiagi) [inline components/collection/drop/MintButton.vue:9]: so when wallet is not connecting => button is not disabled, regardless of `buttonMint.disabled`?

i'm not sure that makes sense
wallet will not be connecting most of the time , no?

**Comment by exezbcz** [comment]
any updates? Please respond to comments or I am closing shortly

**Comment by exezbcz** [comment]
sorry, its blocking and we need this one

**Pull request closed without merging**